servicex/dataset_group.py

```diff
@@ -26,29 +26,47 @@
 # OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 import asyncio
-from typing import Coroutine, List
+from typing import List, Optional
 
-from servicex.models import TransformedResults
+from rich.progress import Progress
+
+from servicex.dataset import Dataset
+from servicex.expandable_progress import ExpandableProgress
+from servicex.models import TransformedResults, ResultFormat
 from make_it_sync import make_sync
 
 
 class DatasetGroup:
-    def __init__(self, datasets: List[Coroutine]):
+    def __init__(self, datasets: List[Dataset]):
```

Comment:
Should this just be a DataSet - so you can have a dataset group as part of a dataset group?

Reply:
Fixed